Library/Formula/lfe.rb

```diff
@@ -22,6 +22,6 @@ def install
   end
 
   test do
-    system bin/"lfe", "-eval", "'(io:format \"42\")'"
+    system bin/"lfe", "-eval", '"(io:format \"~p\" (list (* 2 (lists:foldl #'+/2 0 (lists:seq 1 6)))))"'
```

Comment:
Something about this is causing the test bot to choke.  It might just be the unescaped single-quote in `#'+/2`; try changing that to `#\'+/2` for starters.